bin/.upgrades/versions/v1/upgradeStorage.js

```diff
@@ -57,14 +63,16 @@ async function moveData (context, index, collection, newIndex) {
         delete doc._source._kuzzle_info.deletedAt;
       }
 
-      bulk.push({ index: { _index: newIndex, _id: doc._id } });
+      bulk.push({ create: { _index: newIndex, _id: doc._id } });
```

Comment:
Why changing this ? Aren't we supposed to replace existing data ?

Reply:
It doesn't matter: the [createNewIndex](https://github.com/kuzzleio/kuzzle/pull/1484/files#diff-9fcbc71bf9075c1d138dee6216312ec0R147) function deletes any previously existing index.

As for why I did change this... I made performance tests and thought it had an impact. It doesn't, and I forgot to change it back.... but then again, this makes no difference in our very specific case.